core/src/scene/styleParam.cpp

```diff
@@ -43,14 +34,48 @@ const std::map<std::string, StyleParamKey> s_StyleParamMap = {
     {"outline:width", StyleParamKey::outline_width},
     {"outline:style", StyleParamKey::outline_style},
     {"priority", StyleParamKey::priority},
-    {"repeat_distance", StyleParamKey::repeat_distance},
-    {"repeat_group", StyleParamKey::repeat_group},
     {"size", StyleParamKey::size},
     {"sprite", StyleParamKey::sprite},
     {"sprite_default", StyleParamKey::sprite_default},
     {"style", StyleParamKey::style},
+    {"text:anchor", StyleParamKey::text_anchor},
```

Comment:
I know we came up with this together, but we should refactor this later to avoid adding so many more parameters (if we can).